src/main/resources/RecruitmentApp/FieldsAdministration.xml

```diff
@@ -0,0 +1,382 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<xwikidoc version="1.3" reference="RecruitmentApp.FieldsAdministration" locale="">
+  <web>RecruitmentApp</web>
+  <name>FieldsAdministration</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1557642213000</creationDate>
+  <parent>WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <date>1557642533000</date>
+  <contentUpdateDate>1557642213000</contentUpdateDate>
+  <version>1.1</version>
+  <title>FieldsAdministration</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content>{{velocity output="off"}}
+#set($configDoc = $xwiki.getDocument('RecruitmentApp.Configuration'))
+#set($configClass = $xwiki.getDocument('RecruitmentApp.ConfigurationClass').getxWikiClass())
+#set($discard = $services.localization.use('document', 'RecruitmentApp.AdminTranslations'))
+#set($importConfigDoc = $xwiki.getDocument('RecruitmentApp.ImportConfiguration'))
+#set($importConfigObj = $importConfigDoc.getObject('RecruitmentApp.ImportConfiguration'))

```

Comment:
Are you sure you need the above config lines?

Reply:
I will check this in thorough detail.
These were part of the page previously so I had them kept.